src/EnergyPlus/SetPointManager.hh

```diff
@@ -1051,6 +1002,8 @@ namespace SetPointManager {
 
     int GetMixedAirNumWithCoilFreezingCheck(EnergyPlusData &state, int MixedAirNode);
 
+    Real64 CalcSetPointLinInt(Real64 LowVal, Real64 HighVal, Real64 RefVal, Real64 SetptAtLowVal, Real64 SetptAtHighVal);
+
```

Comment:
As mentioned above, this CalcSetPointLinInt is the alternative of DefineOutsideAirSetPointManager::CalcSetPoint.

Reply:
So you went from a setpoint manager member method to a new global hanging function?  I don't quite understand why.